src/test/java/api/loans/AgedToLostScheduledNoticesProcessingTests.java

```diff
@@ -482,6 +488,11 @@ void patronNoticeForAdjustmentOfFullyPaidLostItemFeeIsCreatedAndProcessed() {
     final JsonObject processingFeeCancellationAction =
       findFeeFineAction(ACTION_TYPE_CANCELLED, PROCESSING_FEE_AMOUNT);
 
+    scaleFields(lostItemFeeRefundAction, "amountAction", "balance");
+    scaleFields(processingFeeRefundAction, "amountAction", "balance");
+    scaleFields(lostItemFeeCancellationAction, "amountAction", "balance");
+    scaleFields(processingFeeCancellationAction, "amountAction", "balance");
```

Comment:
Why do we scale amounts in fee/fine actions? They should remain intact.

Reply:
removed